Applications/APRS/aprs.py

```diff
@@ -260,7 +256,7 @@ def sendPositions(stations, socket):
                 try:
                     socket.sendall(positionString)
 
-                except socket.error as e:
+                except IOError as e:
```

Comment:
`socket.error` is a [child of IOError](https://docs.python.org/2/library/exceptions.html#exceptions.IOError). This means IOError is a little more broad and sweeping. Additionally for some reason `socket.error` was not being found [when used in aprs.py](https://github.com/FaradayRF/Faraday-Software/issues/91#issuecomment-281220467). Using IOError fixed this as it's a built-in exception. I can investigate more as to why the socket module wasn't letting me do this but I figured this was an acceptable fix.